<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPreviewSettingsDialog</class>
 <widget class="QDialog" name="EditPreviewSettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Preview Settings</string>
  </property>
  <widget class="QLineEdit" name="deviceNameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Screen Width</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>75</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Screen Height</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="dpiSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="diagonalSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>145</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dpiRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>DPI</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="diagonalRadioButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>97</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Diagonal</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
